### ISSUE TITLE
Lost streak

### ISSUE THREAD

**Issue by matteing** [getmakerlog/web#143]
---

Author Name: **Anonymous** (Anonymous)
Original Redmine Issue: 152, https://pm.mattei.dev/issues/152
Original Date: 2019-07-27

---

I had some weird connection issues and just logged a task, and now my streak reset to 2, rather than 18 or 19 (I can't remember).

@jessesutherland

Thanks! Love the website!

**Comment by ivmirx** [comment]
I lost mine too: was checking daily but a few days ago a rest icon appeared and today it went down from 11 to 4.

![Captura de Pantalla 2020-10-26 a la(s) 18 42 58](https://user-images.githubusercontent.com/10554114/97232066-46dace00-17bb-11eb-8aae-0b8681232336.png)

**Comment by matteing** [comment]
@ivanmir did you reply to this?

**Comment by ivmirx** [comment]
Yep, but quickly realized it was my fault because I deleted one task from my log recently. Though it would be nice to keep placeholders for deleted tasks, so the counter doesn't modify retroactively.

**Comment by matteing** [comment]
What’s your username? I can fix it for you.

On Oct 26, 2020, at 8:45 PM, Ivan Mir <notifications@github.com<mailto:notifications@github.com>> wrote:



Yep, but quickly realized it was my fault because I deleted one task from my log recently. Though it would be nice to keep placeholders for deleted tasks, so the counter doesn't modify retroactively.

**Comment by ivmirx** [comment]
The same, [@ivmirx](https://getmakerlog.com/@ivmirx)

**Comment by matteing** [comment]
Done

On Oct 26, 2020, at 8:49 PM, Ivan Mir <notifications@github.com<mailto:notifications@github.com>> wrote:



The same, @ivmirx<https://getmakerlog.com/@ivmirx>

**Comment by ivanmir** [comment]
> @ivanmir did you reply to this?

I have nothing to do with this issue. Maybe it is another Ivan...

**Comment by matteing** [comment]
haha sorry, I wrote the wrong username @ivanmir
